plugins/lookup/aws_secret.py

```diff
@@ -58,6 +63,9 @@
 '''
 
 EXAMPLES = r"""
+ - name: lookup secretsmanager secret in the current region
+   debug: msg="{{ lookup('aws_secret', '/path/to/secrets', bypath=true) }}"
```

Comment:
Examples must use FQCN. (It seems that the other examples in this file also don't do that CC @tremble @jillr)
```suggestion
   debug: msg="{{ lookup('amazon.aws.aws_secret', '/path/to/secrets', bypath=true) }}"
```

Reply:
I went ahead added this to the other examples as well.